src/app/tabs/debugger/debuggerUI/vmDebugger/utils/SolidityTypeFormatter.js

```diff
@@ -55,16 +58,20 @@ function extractData (item, parent, key) {
 }
 
 function fontColor (data) {
-  var color = '#124B46'
+  var color = 'var(--primary)'
   if (data.isArray || data.isStruct || data.isMapping) {
-    color = '#847979'
-  } else if (data.type.indexOf('uint') === 0 ||
-              data.type.indexOf('int') === 0 ||
-              data.type.indexOf('bool') === 0 ||
-              data.type.indexOf('enum') === 0) {
-    color = '#0F0CE9'
+    color = 'var(--info)'
+  } else if (
```

Comment:
Create a variable or a function for this `if` statement that explain what it's. Something like : 
```
function isType(data, types) {
  return types.some(type => data.type[0] === type);
}
...
else if (isType(data, ['uint', 'int', 'bool', 'enum'])) { ... }
```